db/migrate/20160513081219_rename_old_table_to_new_table.rb

```diff
@@ -0,0 +1,5 @@
+class RenameOldTableToNewTable < ActiveRecord::Migration
```

Comment:
The name for this file could be a lot better. For now it's fine. But in the future make it more descriptive.